Question3_LongestIncreasingSequence.java

```diff
@@ -0,0 +1,41 @@
+import java.util.Arrays;
+/**
+ * This class is designed to find longest increasing sequence in an array
+ * @author Gaurav Tak
+ *
+ */
+public class Question3_LongestIncreasingSequence {
+	/**
+	 * This method finds the longest increasing sequence
+	 * @param input
+	 * @return Longest Incresing Sequence as integer array
+	 */
+	public int[] longestSequence(int input[]) {
+		 int lis,start,next,lisLength=0,seqLength=0;
+		 lis=0;		//index of the longest increasing sequence
+		 start=0;	//the index of start of current increasing sequence
+		 next=1;	//the index of next element of current element
+		 for(int i=1;i<input.length;i++){
```

Comment:
1) Again can check for null input array
2) Can have better naming convention for lis